examples/full-example/app.js

```diff
@@ -6,10 +6,18 @@ var ReactDOMServer = require('react-dom/server');
 var express = require('express');
 var logger = require('morgan');
 var MendelMiddleware = require('mendel-middleware');
+var cache = true;
+
 if (process.env.NODE_ENV !== 'production') {
     MendelMiddleware = require('mendel-development-middleware');
+    cache = false;
+}
+
+if (String(process.env.CACHE) === 'false') {
```

Comment:
It is a very contrived example, not an API supported by Mendel... but I'll change it anyway.